### ISSUE TITLE
Piping the output of agent list command only returns up to 20 entries

### ISSUE THREAD

**Issue by adrysn** [lablup/backend.ai#677]
**Describe the bug**
I tried to get every Agent ID with the CLI command. There were over 30 agents in the server farm and `backend.ai admin agent list` command just worked well by returning every Agent list with the pagination. However, when I pipe the result with commands like `awk` or `wc -l`, I only got 20 entries.

**To Reproduce**
1. First, since a test cluster usually does not have >20 Agents, I created 50 fake `TERMINTED` agents.
    ```shell
    for i in {1..50}; do
        slots='{"cpu": "4", "mem": "34359738368", "cuda.shares": "0"}'
        fields='"id", status, region, scaling_group, available_slots, occupied_slots, addr, version, compute_plugins, architecture'
        values="'i-test-${i}', 'TERMINATED', 'local', 'default', '${slots}'::jsonb, '${slots}'::jsonb, 'tcp://1.2.3.4:6001', '22.09.0b4', '{}'::jsonb, 'x86_64'"
        insert_query="INSERT INTO agents ($fields) VALUES ($values);"
        echo $insert_query
        # Use your own DB's container name or ID.
        docker exec -it main_backendai-half-db_1 psql -U postgres -d backend -c "$insert_query"
    done
    ```
2. Confirm that `backend.ai admin agent list -s TERMINATED` command returns >=50 Agents (including the real ones). 
    <img width="833" alt="image" src="https://user-images.githubusercontent.com/7539358/185955343-4e78f48d-8c49-4119-8318-5e7e45a5269b.png">
3. Now, pipe the result with any command. For example, I used `awk` to get the Agent's IDs only: `backend.ai admin agent list -s TERMINATED | awk '{print $1}'`. Then, I got just 20 Agent IDs:
    <img width="461" alt="image" src="https://user-images.githubusercontent.com/7539358/185955841-90a11171-e243-4f3e-b5f4-c50bf85ce972.png">
4. `wc -l` command confirms again (2 for header rows):
    <img width="387" alt="image" src="https://user-images.githubusercontent.com/7539358/185956125-92aafb0d-efcc-4855-8ba2-f46f75b2821f.png">

**Expected behavior**
Piping the output of the Agent list command should return every item.

**Client:**
 - OS: CentOS 7.9
 - Python version: 3.10.5

**Server:**
 - OS: CentOS 7.9
   - Output of `uname -snrmpio`: `Linux asto1 3.10.0-1160.21.1.el7.x86_64 x86_64 x86_64 x86_64 GNU/Linux`
 - Backend.AI version: 22.09.0b4
 - Python version: 3.10.5
 - Installation method: on-premise

**Additional context**
Other commands like `session list` also shows the same issue:
<img width="390" alt="image" src="https://user-images.githubusercontent.com/7539358/185958938-400afdd7-fc45-4e58-9ad7-50f2916bd615.png">

If you want to delete the test Agents, just execute the script:
```shell
for i in {1..50}; do
    delete_query="DELETE FROM agents WHERE \"id\" = 'i-test-${i}';"
    echo $delete_query
    # Use your own DB's container name or ID.
    docker exec -it main_backendai-half-db_1 psql -U postgres -d backend -c "$delete_query"
done
```

**Comment by ChangHoon-Sung** [comment]
I found that if the stdout is not a tty (like pipe) or page_size is given, then client.console.print_pagenated_list() sets page_size to the given value or 20 as default. 
in this case, it prints the results without repeating fetch. (issue caused)

https://github.com/lablup/backend.ai/blob/main/src/ai/backend/client/output/console.py#L114-L154

However, it seems that the repeating fetch function already exists in 'stdout is a tty' case as infinite_fetch(). I simply checked that it works on 'not a tty' case too.

![ScreenShot 2023-01-06 at 18 40 54](https://user-images.githubusercontent.com/37946887/210974370-9d3a9267-6e4a-49ce-a73e-175c5cc1e9e7.png)

@achimnol Could I apply the infinite_fetch() on 'stdout is not a tty' case too?
